src/mapbox_logo.js

```diff
@@ -0,0 +1,46 @@
+
+        return this._update();
+    },
+
+    _update: function() {
+        if (!this._map) { return this; }
+
+        this._container.innerHTML = this._mapboxLogo;
+
+        return this;
+    }
+});
+
+module.exports.MapboxLogo = MapboxLogo;
+
+module.exports.MapboxLogo = function(options) {
```

Comment:
`module.exports.mapboxLogo = function(options)` (initial lowercase for factory functions).